sdk/src/builder.rs

```diff
@@ -193,6 +193,12 @@ pub struct Builder {
     #[serde(flatten)]
     pub definition: ManifestDefinition,
 
+    /// Optional remote URL for the manifest
+    pub remote_url: Option<String>,
+
+    // If true, the manifest store will not be embedded in the asset on sign
+    pub no_embed: bool,
+
```

Comment:
Seems like we'd eventually get to the `DestOption` bitmask version of this. Should we do that now? I know we don't have the sidecar APIs figured out yet, so maybe this is a problem to solve later on?

Reply:
DestOption doesn't work here because we need to specify the remote_url. I'm not sure this API is final. Things are different with streams and I'm working on that. Basically you always get signed c2pa_manifest you can write to the cloud or a sidecar. But the output file may need to have a url, a manifest, or both, and we also have to strip out manifests and urls when needed.